extensions/ui/org.eclipse.smarthome.ui.basic/snippets/switch.html

```diff
@@ -2,13 +2,17 @@
 	<span class="mdl-form__icon">
 		<img data-icon="%category%" src="../icon/%category%?state=%state%&format=%icon_type%" />
 	</span>
-	<div class="mdl-form__label">
+	<span %labelstyle% class="mdl-form__label">
 		%label%
-	</div>
+	</span>
+	<span %valuestyle% class="mdl-form__value mdl-form__value--group">
```

Comment:
Ok, makes sense. I am starting to understand the philosophy with CSS and less.
It looks like the font looked less bold than for a normal text item or maybe the color was less black. Any hint on this ?